### PR TITLE
[WIP] Live editing

### PR DESCRIPTION
**Pull request by megawac**
Very much WIP

Semi dependent on #129 

Uses CodeMirror to do live editing (currently doesn't compile [help! @jdaudier]). Going for something similar to http://www.ractivejs.org/ demos

-------------------

How should scripts be sandboxed `iframe` -- something like https://github.com/jterrace/js.js?

### CODE REVIEW

File: src/stylesheets/marionette.scss [hunk 1]
```diff
@@ -15,6 +15,9 @@
 @import "downloads";
 @import "back_to_top_button";
```
Comment (megawac) [inline src/stylesheets/marionette.scss]: Haven't bothered removing the prettify stuff yet btw

**Comment by samccone** [comment]
@megawac mind rebasing off of master

**Comment by megawac** [comment]
This has a dependency on e4bcaa1

**Comment by samccone** [comment]
that was merged @megawac

**Comment by megawac** [comment]
Done, think I'm just going to render on iframe when I get sometime to finish this up (prob end of the week - feel free to take over this branch if you want to/work on it elsewhere). If someone wants to point out what I messed up with for the jade file I'd appreciate it

**Comment by samccone** [comment]
awesome :)

File: src/sections/_code_section.jade [hunk 2]
```diff
@@ -1,22 +1,23 @@
...
-      pre.prettyprint
-        code.
-          var SingleLink = Marionette.ItemView.extend({
-            tagName: "li",
-            template: _.template("&lt;a href='<%-path%>'><%-path%>&lt;/a>")
-          });
+section#code-samples
+  .example-tab
+    input#(type="radio", name="example-index", value="#example-index-0") CollectionView
+    input#(type="radio", name="example-index", value="#example-index-1") Applications
```
Comment (samccone) [inline src/sections/_code_section.jade]: the # here is invalid syntax

**Comment by samccone** [comment]
hey @megawac 
looks like there is a great reason to finish this one out 

screen shot 2015-02-10 at 9 11 45 pm


A fairly significant portion of our visitors expect the ability to use the code area.
Let me know if you need any help

**Comment by megawac** [comment]
Yeah, will you be around this weekend? - I won't have any time for this til then

**Comment by samccone** [comment]
yep I will be, we can pair on it or whatever then

**Comment by samccone** [comment]
Closing for now... we can reopen when people have some time to work on this.

**Pull request closed without merging**